### PR TITLE
Sqlite3

### PR DESCRIPTION
**Pull request by volodyapopyk**
my last db homework

### CODE REVIEW

File: dbtask2 [hunk 1]
```diff
@@ -0,0 +1 @@
+SELECT purchase.id, purchase.date, first_name, last_name FROM purchase, users
```
Comment (acman) [inline dbtask2:1]: According to task `INNER JOIN` should be used here, like 
```suggestion
SELECT p.id, p.date, first_name, last_name
FROM users INNER JOIN purchases p on users.id = p.user_id;
```

File: dbtask3 [hunk 2]
```diff
@@ -0,0 +1,5 @@
+SELECT 'http://users.'||users.id||'/', first_name, last_name,books.title
```
Comment (acman) [inline dbtask3:1]: nit: it was mistake in task [thread](https://robotdreamscc.slack.com/archives/C04BLF231NX/p1676828262891219) So only `users.id` can be used here and below

File: dbtask3 [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+FROM users
+LEFT JOIN purchase on users.id = purchase.user_id
```
Comment (acman) [inline dbtask3:3]: nit: `INNER JOIN` can be used here as well or probably I have not enough testing data

File: dbtask5 [hunk 4]
```diff
@@ -0,0 +1,4 @@
+SELECT books.author ,count(book_id) as amount
+FROM books
+JOIN purchase
+WHERE books.author = 'Rowling' AND book_id > 3
```
Comment (acman) [inline dbtask5:4]: why `book_id > 3` :)

File: dbtask7 [hunk 5]
```diff
@@ -0,0 +1,6 @@
+SELECT books.author , COUNT(book_id) as number_of_purchases,
+       iif(SUM(price) <= 900, '0', SUM(price) ) AS suma
+FROM  books
+LEFT JOIN  purchase ON purchase.book_id = books.id
+GROUP BY books.author
+order by books.author
```
Comment (acman) [inline dbtask7:6]: I didn't get why there is so complex condition
```suggestion
SELECT books.author, COUNT(purchases.book_id) AS count_purchases, SUM(books.price) AS sales_amount
FROM purchases
INNER JOIN books ON purchases.book_id = books.id
GROUP BY books.author
```

**Review by acman — CHANGES_REQUESTED** [review]

**Review by vhreb — APPROVED** [review]
👍

**Pull request merged**